### PR TITLE
Removing duplicates from devfile catalog test

### PR DESCRIPTION
**Pull request by prietyc123**
**What type of PR is this?**

/kind cleanup

**What does does this PR do / why we need it**:

Removes the duplicates from test script

**Which issue(s) this PR fixes**:

Fixes - Part of #3385

**PR acceptance criteria**:

- [ ] Unit test 

- [x] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer**:

`make test-cmd-devfile-catalog` should pass.

### CODE REVIEW

File: tests/integration/devfile/cmd_devfile_catalog_test.go [hunk 1]
```diff
@@ -103,9 +102,7 @@ var _ = Describe("odo devfile catalog command tests", func() {
 	Context("When executing catalog describe component with a component name with multiple components", func() {
 		It("should print multiple devfiles from different registries", func() {
 			helper.CmdShouldPass("odo", "registry", "add", registryName, addRegistryURL)
-			output := helper.CmdShouldPass("odo", "registry", "list")
```
Comment (prietyc123) [inline tests/integration/devfile/cmd_devfile_catalog_test.go:106]: We have a separate test file for devfile registry and checking registry commands in catalog file just duplicating the steps. As we are adding the same registry url `https://github.com/odo-devfiles/registry` in [devfile-regsitry](https://github.com/openshift/odo/blob/master/tests/integration/devfile/cmd_devfile_registry_test.go#L61-L63) So removing it from here.

File: tests/integration/devfile/cmd_devfile_catalog_test.go [hunk 2]
```diff
@@ -84,7 +84,6 @@ var _ = Describe("odo devfile catalog command tests", func() {
 				"java-springboot",
 				"java-quarkus",
 			})
-			helper.CmdShouldPass("odo", "registry", "delete", "fake", "-f")
```
Comment (amitkrout) [inline tests/integration/devfile/cmd_devfile_catalog_test.go:87]: Test script is creating the fake registry, so script should clean it up. Please add it back.

Comment (prietyc123) [reply tests/integration/devfile/cmd_devfile_catalog_test.go]: Done

**Pull request merged**